symfony/src/Dto/CustomerDTO.php

```diff
@@ -6,15 +6,11 @@
 use App\Entity\Customer;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\Validator\Constraints as Assert;
-use Symfony\Component\Validator\Constraints\EmailValidator;
 
 final class CustomerDTO
 {
     /**
-     * @Assert\Type(
-     *     type="integer",
-     *     message="The value {{ value }} is not a valid {{ type }}."
-     * )
+     * @Assert\Type("integer")
      */
     public $id;
```

Comment:
If properties defined as public in the class there is no need to create getters of setter, as you can always get the value from the property itself.